tests/acceptance/CreateContentModelMediaFieldCest.php

```diff
@@ -8,6 +8,61 @@ public function _before( \AcceptanceTester $i ) {
 		$i->amOnWPEngineEditContentModelPage( $content_model['slug'] );
 	}
 
+	/**
+	 * Ensure a user can add a Media field to the model with a repeatable property.
+	 */
+	public function i_can_create_a_content_model_media_repeatable_field( AcceptanceTester $i ) {
+		$i->loginAsAdmin();
+		$content_model = $i->haveContentModel( 'Candy', 'Candies' );
+		$i->amOnWPEngineEditContentModelPage( $content_model['slug'] );
+
+		$i->click( 'Media', '.field-buttons' );
+		$i->wait( 1 );
+		$i->fillField( [ 'name' => 'name' ], 'positionxyz' );
+
+		// Set the Input Type to “multiple lines” instead of “Single line”.
+		$i->click( '.is-repeatable' );
+
+		// Save the field.
+		$i->click( 'button[data-testid="edit-model-update-create-button"]' );
+		$i->wait( 1 );
+
+		// Create a new Candies entry.
+		$i->amOnPage( '/wp-admin/edit.php?post_type=candy' );
+		$i->click( 'Add New', '.wrap' );
+		$i->wait( 1 );
+
+		// Check that the first input in the index of available list items is rendering indicating a multiple being set.
+		$i->seeElement(
+			'input',
+			[ 'name' => 'atlas-content-modeler[candy][positionxyz][0]' ]
+		);
+	}
+
+	/**
+	 * Ensure a user cannot save a Media field with a required repeatable property.
+	 */
+	public function i_cannot_save_empty_required_media_repeatable_field( AcceptanceTester $i ) {
+		$i->loginAsAdmin();
+		$content_model = $i->haveContentModel( 'Candy', 'Candies' );
+		$i->amOnWPEngineEditContentModelPage( $content_model['slug'] );
+
+		$i->click( 'Media', '.field-buttons' );
+		$i->wait( 1 );
+		$i->fillField( [ 'name' => 'name' ], 'positionxyz' );
+
+		$i->click( '.is-repeatable' );
+		$i->click( '.is-required' );
+		$i->click( 'button[data-testid="edit-model-update-create-button"]' );
+		$i->wait( 1 );
+
+		// Create a new Candies entry.
+		$i->amOnPage( '/wp-admin/edit.php?post_type=candy' );
+		$i->click( 'Add New', '.wrap' );
+		$i->wait( 1 );
+		$i->seeElement( 'input[name="atlas-content-modeler[candy][positionxyz][0]"]:invalid' );
```

Comment:
I think this test is failing because no field with a name of `atlas-content-modeler[candy][positionxyz][0]` exists until media has been selected from the media picker. (We only render the repeatable fields once a user has chosen media.)

If you apply the suggestion from Slack to render a hidden required input field when the user has not selected any media, you could then adapt this test to check for that field (by dropping the `[0]`):

```suggestion
		$i->seeElement( 'input[name="atlas-content-modeler[candy][positionxyz]"]:invalid' );
```